xwiki-platform-core/xwiki-platform-oldcore/src/main/java/com/xpn/xwiki/web/DeleteAction.java

```diff
@@ -317,4 +339,14 @@ private Job startDeleteJob(EntityReference entityReference, XWikiContext context
             throw new XWikiException(String.format("Failed to schedule the delete job for [%s]", entityReference), e);
         }
     }
+
+    protected DocumentReferenceResolver<String> getCurrentStringDocumentReferenceResolver()
```

Comment:
Same, this is become visible to users (who extend DeleteAction). Best to avoid it if possible and needs `@since` and `@Unstable`.